docker-compose.yml

```diff
@@ -93,12 +89,24 @@ services:
       start_period: 120s
     command:
       - uwsgi --ini /data/olympia/docker/uwsgi.ini
+    volumes:
+      # Don't mount generated files. They only exist in the container
+      # and would otherwiser be deleted by mounting the cwd volume above
+      - data_static_build:/data/olympia/static-build
+      - data_site_static:/data/olympia/site-static
 
   nginx:
     image: nginx
+    environment:
+      - STATIC_ROOT=/data/olympia
```

Comment:
We have a Django setting called `STATIC_ROOT` already that points to a different directory, so that'd be confusing... And on top of that, below, you're using that as the root for storage as well, not just static files...

There is also a setting called just `ROOT` which does match, as its the root directory containing the app, so maybe rename that env variable to `ROOT` for consistency ?

Reply:
This is for nginx.. it should not interact with anything in django land, but happy to rename it. This is where we are mounting the entire root directory so STATIC_ROOT/storage and STATIC_ROOT/static as well as STATIC_ROOT/site-static all point to the same and correct places in the container.

Reply:
Yeah I think just calling this `ROOT` for consistency would be enough.